### ISSUE TITLE
Accessible icon information

### ISSUE THREAD

**Issue by ianbjacobs** [w3c/secure-payment-confirmation#127]
For users with some visual disabilities, the icon may not be accessible. Would it make sense to support an "alt" string as input (e.g., that the browser can make available in the UX or to assistive technologies)?

**Comment by rsolomakhin** [comment]
I would consider the instrument display name to be this alt text.

**Comment by ianbjacobs** [comment]
The instrument string does not convey the same information. A card number does not tell you the issuing institution (for example).

**Comment by rsolomakhin** [comment]
An alt text for the screen reader could make sense if the icon's intention is to convey entirely different information from its display name.

**Comment by ianbjacobs** [comment]
@RealJoshue108,

My expectation is that the icon will tell the user about the issuer of the payment instrument, for example "Mastercard" or "Barclays". Thus, the alt text might be read as "Mastercard ***1234".

**Comment by stephenmcgruer** [comment]
I would love to hear from e.g. @jcemer-stripe , @davordavidovic, or other users of SPC on whether having alt-text for the icon would provide **additional** explanation value on top of the displayName, or whether its just visual 'eye-candy'.

For example, if most displayNames are `NetworkName ****1234` anyway, and the alt text for the icon would also be e.g. `NetworkName icon`, then to me not much value is added. On the other hand, if displayNames are often something like `Stephen's card ****1234` then the icon is distinct and an alt-text makes a lot of sense.

**Comment by jcemer-stripe** [comment]
In the context of card payment, the icon is used more as a 'eye-candy'. I don't see much value of adding an alt-text for cards but it could be a nice touch for other use cases.

**Comment by rsolomakhin** [comment]
> eye-candy

Do you mean to say that it's not important for accessibility / for understanding the context of the payment?

**Comment by jcemer-stripe** [comment]
> > eye-candy
> 
> Do you mean to say that it's not important for accessibility / for understanding the context of the payment?

I cannot make that strong statement but I think that for card payments the icon is pretty much the card brand which is in general already present in the display name.

**Comment by ianbjacobs** [comment]
Are there use cases where one might do something like this:

[Bank icon] Mastercard *****1234

That seems to be the main use case.

**Comment by ianbjacobs** [comment]
@RealJoshue108, please see pull request #162, which recommends using the displayName to ensure the accessibility of the icon (rather than adding another alt value). According to comments above, the displayName sounds like it will suffice (or can be made to do so). thanks!

**Comment by michael-n-cooper** [comment]
APA is ok with how this issue was handled. Thx!

**Comment by ianbjacobs** [comment]
Thank you @michael-n-cooper!
